src/util/rest/mapping/synced.js

```diff
@@ -0,0 +1,8 @@
+const isEqual = require('lodash.isequal');
+const { getIndexVersions } = require('../index/versions');
+
+module.exports = async (call, versions, idx) => {
+  const localVersions = Object.keys(versions.get(idx));
+  const remoteVersions = await getIndexVersions(call, idx);
+  return isEqual(localVersions, remoteVersions);
```

Comment:
we only want to check that every local version exists remote (since that's what the sync command does)

Reply:
We might want to have a "pruned" function as well to match what the prune command does?